substrate/runtime/staking/src/address.rs

```diff
@@ -21,14 +21,16 @@ use super::{Member, Slicable, As, Input};
 
 /// A vetted and verified extrinsic from the external world.
 #[derive(PartialEq, Eq, Clone)]
-#[cfg_attr(feature = "std", derive(Serialize, Debug))]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize, Debug))]
 pub enum Address<AccountId, AccountIndex> where
  	AccountId: Member,
  	AccountIndex: Member,
 {
 	/// It's an account ID (pubkey).
+	#[cfg_attr(feature = "std", serde(deserialize_with="AccountId::deserialize"))]
```

Comment:
why?

Reply:
workaround for one of the rustc issues that causes issues with serde derives.